overlord/devicestate/firstboot_test.go

```diff
@@ -216,9 +224,6 @@ func (s *FirstBootTestSuite) TestPopulateFromSeedOnClassicNoSeedYaml(c *C) {
 	assertsChain := s.makeModelAssertionChain(c, "my-model-classic", nil)
 	s.WriteAssertions("model.asserts", assertsChain...)
 
-	err = os.Remove(filepath.Join(dirs.SnapSeedDir, "seed.yaml"))
```

Comment:
I don't quite follow why we no longer need to remove the seed.yaml from this test run, couldn't that file being left around still conflict with other firstBoot16Suite tests?

Reply:
ah perfect thanks for explaining